app/models/CustomsOfficeList.scala

```diff
@@ -19,9 +19,11 @@ package models
 import models.reference.CustomsOffice
 import play.api.Logging
 import play.api.http.Status.{NOT_FOUND, OK}
+import play.api.libs.json.{JsArray, JsError, JsSuccess, Reads}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.select.SelectItem
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
 
-case class CustomsOfficeList(customsOffices: Seq[CustomsOffice]) {
+case class CustomsOfficeList(customsOffices: Seq[CustomsOffice]) extends Selectable {
```

Comment:
Does this need to extend Selectable? It’s been a while since that trait was made but I think that refers to an individual “thing” (a country, a customs office etc.)

Reply:
Yep you're right, I had forgotten to remove that. No need for this to extend Selectable.